sdk/react/src/lib/fragment-components/a-time-picker.tsx

```diff
@@ -0,0 +1,360 @@
+                                    value="${step.value}"
+                                    text="${step.text}">
+                                        ${step.text}
+                                    </option>`
+                                )).join('\n')}
+                            </ibm-timepicker-select>
+                        </ibm-timepicker>`;
+                }
+            },
+            v10: {
+                inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = "${json.value}";`,
+                outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+                imports: ['TimePickerModule', 'TimePickerSelectModule'],
+                code: ({ json }) => {
+                    return `<ibm-timepicker
+                        ${angularClassNamesFromComponentObj(json)}
+                        label="${json.label}"
+                        ${json.light ? '[theme]="light"' : '[theme]="dark"'}
+                        [invalid]="${json.invalid}"
+                        [placeholder]="'${json.placeholder}'"
+                        [size]="'${json.size}'"
+                        [hideLabel]="${json.hideLabel}"
+                        [invalidText]="'${json.invalidText}'"
+                        (valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)"
+                        [value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+                        [disabled]="${json.disabled}">
+                            <ibm-timepicker-select ${json.light
+                                ? '[theme]="light"'
+                                : '[theme]="dark"'
+                            }
+                            [disabled]="${json.disabled}"
+                            display="inline">
+                                <option selected value="AM">AM</option>
+                                <option value="PM">PM</option>
+                            </ibm-timepicker-select>
+                            <ibm-timepicker-select ${json.light
+                                ? '[theme]="light"'
+                                : '[theme]="dark"'
+                            }
+                            [disabled]="${json.disabled}"
+                            display="inline">
+                                ${json.items.map((step: any) => (`<option
+                                    value="${step.value}"
+                                    text="${step.text}">
+                                        ${step.text}
+                                    </option>`
+                                )).join('\n')}
+                            </ibm-timepicker-select>
+                        </ibm-timepicker>`;
+                }                                
+            }
+		},
+		react: {
```

Comment:
This is indented.

I recommend you run the linter... quite a few cases where you're using double quotes instead of single... Linter should automatically fix.